apps/api/src/modules/user/user.controller.ts

```diff
@@ -16,39 +15,43 @@ export const UserController: RouterImplementation<typeof contract.user> = {
     throw new InternalServerErrorException('Not implemented');
   },
   getUsers: async ({ query: { userIds, search, skip, take } }) => {
-    prisma.user.findMany();
-    // const users = await Service.getManyUsers({ userIds, search, skip, take });
-    return {
-      status: 200,
-      body: {
-        users: [
-          {
-            id: 1,
-            user: 'User 1',
-            email: 'user1@example.com',
-            description: 'Content 1',
-          },
-          {
-            id: 2,
-            user: 'User 2',
-            email: 'user2@example.com',
-            description: 'Content 2',
-          },
-          {
-            id: 3,
-            user: 'User 3',
-            email: 'user3@example.com',
-            description: 'Content 3',
-          },
-          {
-            id: 4,
-            user: 'User 4',
-            email: 'user4@example.com',
-            description: 'Content 4',
-          },
-        ],
-        total: 2,
-      },
-    };
+    try {
+      const users = (await UserService.getManyUsers({ userIds, search, skip, take })) as Array<{
+        uid?: string;
+        id?: number;
+        user: string;
+        email: string;
+        dob?: string;
+        pref_name?: string;
+        first_name?: string;
+        last_name?: string;
+        payment_method?: string;
+        subscription_plan?: string;
+      }>;
+
```

Comment:
In controller, we don't need try catch for this one and casting since implementing the service will type-safely infer the type from prisma
```suggestion
    const users = await UserService.getManyUsers({ userIds, search, skip, take })
```